test/testers.jl

```diff
@@ -546,4 +546,22 @@ end
         test_frule(f_notimplemented, randn(), randn())
         test_rrule(f_notimplemented, randn(), randn())
     end
+
+    @testset "@maybe_inferred" begin
+        f_noninferrable(x) = Ref{Real}(x)[]
+
+        @test @maybe_inferred(identity(1)) == 1
+        errors("not inferrable") do
```

Comment:
`errors` returns `true` if the error occured.
```suggestion
       @test errors("not inferrable") do
```

Reply:
Ah, makes sense!